### PR TITLE
Add Eltex fingerprints

### PR DESCRIPTION
**Pull request by tsellers-r7**
## Description
Add fingerprints for various Eltex branded network equipment.

Vendor website: https://eltex-co.com/catalog/

All existing CPE currently issued by NIST.
```
h:eltex-co:ntp-2
h:eltex-co:ntp-rg-1402g
h:eltex:esp-200
o:eltex-co:ntp-2_firmware
o:eltex-co:ntp-rg-1402g_firmware
o:eltex:esp-200_firmware
```

`sofia-sip` reference: https://github.com/freeswitch/sofia-sip

Verification of TAU firmware numbering scheme: https://eltex-co.com/catalog/tau-16-ip-en.php
Verification of sofia-sip version numbering scheme: https://github.com/freeswitch/sofia-sip/releases

## Motivation and Context
Improved coverage

## How Has This Been Tested?
`rspec`

## Types of changes
Fingerprint additions

## Checklist:

- [x] I have updated the documentation accordingly (or changes are not required).
- [x] I have added tests to cover my changes (or new tests are not required).
- [ ] All new and existing tests passed.

### CODE REVIEW

File: xml/html_title.xml [hunk 1]
```diff
@@ -3830,4 +3830,59 @@
     <param pos="0" name="hw.family" value="Vigor"/>
   </fingerprint>
 
+  <!-- Specific fingerprints to enable CPE generation -->
```
Comment (mkienow-r7) [inline xml/html_title.xml:3833]: Minor: It might be helpful to add "Eltex fingerprints" to the comment.

**Review by mkienow-r7 — CHANGES_REQUESTED** [review]
Please ensure that the parameters are ordered: `service`, `os`, `hw`.

File: xml/sip_user_agents.xml [hunk 2]
```diff
@@ -618,4 +618,81 @@
...
+    <param pos="0" name="os.device" value="VoIP Gateway"/>
+    <param pos="0" name="hw.vendor" value="Eltex"/>
+    <param pos="1" name="hw.product"/>
+    <param pos="3" name="hw.serial_number"/>
+    <param pos="0" name="hw.device" value="VoIP Gateway"/>
+  </fingerprint>
+
+  <fingerprint pattern="^(TAU-\d[\d.IP]+)/([\d.]+) SN/(VI\w+) (?:SHA/[0-9a-f]+ )?sofia-sip/([\d.]+)$">
+    <description>Eltex TAU model VoIP gateway - with serial number and sofia version</description>
+    <example hw.product="TAU-8.IP" hw.serial_number="VI12345678">TAU-8.IP/2.3.0 SN/VI12345678 sofia-sip/1.12.10</example>
```
Comment (tsellers-r7) [inline xml/sip_user_agents.xml:637]: Verification of TAU firmware numbering scheme: https://eltex-co.com/catalog/tau-16-ip-en.php
Verification of `sofia-sip` version numbering scheme: https://github.com/freeswitch/sofia-sip/releases

Comment (mkienow-r7) [reply xml/sip_user_agents.xml:637]: The character class will allow some combinations that do not appear to be valid products, for example, `TAU-8.I` or `TAU-8.P`. Should the `.IP` be outside of the character class?
```
(TAU-\d[\d.IP]+)
```

Comment (mkienow-r7) [reply xml/sip_user_agents.xml:637]: An outlier from the one or two digits followed by ".IP" is the [TAU-32M.IP](https://eltex-co.com/catalog/tau-32m-ip-en.php).

Comment (tsellers-r7) [reply xml/sip_user_agents.xml:637]: RE: the business routers: Their primary function is as a VoIP gateway and have VoIP gateway in the product description. 

RE: TAU-32M.IP - I don't have a SIP User Agent example for this product. There are quite a few banners yet to be added for this vendor so I'd expect this to be added in the future if it is observed. I'd recommend re-fingerprinting existing data or releasing a Recog version and pulling metrics from future Sonar study.

Comment (mkienow-r7) [reply xml/sip_user_agents.xml:637]: I spoke with @tsellers-r7 and 
1. the business routers with VoIP support should be included
2. the character class concern is "it's low risk", but could regex could be updated
Tom doesn't have time to address these so I'll process the PR as-is and follow-up with another PR.

**Review by mkienow-r7 — APPROVED** [review]

**Comment by mkienow-r7** [comment]
Thank you for the contribution @tsellers-r7!

**Pull request merged**